File: src/net/eduard/permissions/command/PermissionsUserSetSuffixCommand.java

package net.eduard.permissions.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.modules.Mine;
import net.eduard.api.lib.manager.CommandManager;
import net.eduard.api.lib.modules.FakePlayer;
import net.eduard.permissions.EduPermissions;
import net.eduard.permissions.manager.PermissionsManager;
import net.eduard.permissions.manager.PermissionsPlayer;

public class PermissionsUserSetSuffixCommand extends CommandManager {

	public PermissionsUserSetSuffixCommand() {
		super("setsuffix", "definirsufixo");
		setUsage("/permissions user setsuffix <player> <permission>");
		setDescription("Definir o sufixo do jogador");
	}

	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {

		PermissionsManager manager = EduPermissions.getInstance().getManager();
		if (args.length < 4) {
			sendUsage(sender);
		} else {
			String nome = args[2];
			String suffixo = Mine.toChatMessage(args[3]);
			PermissionsPlayer user = manager.getPlayer(new FakePlayer(nome));

//			PermissionsGroup group = manager.getGroup(nome);
			if (user != null) {
				user.setSuffix(suffixo);
				sender.sendMessage(EduPermissions.getInstance().message("player-set-prefix")
						.replace("$player", "" + nome).replace("$prefix", suffixo));
			} else {
				sender.sendMessage(
						EduPermissions.getInstance().message("player-not-exists").replace("$player", "" + nome));
			}
//			
		}

		return true;
	}

}

File: src/main/java/net/eduard/permissions/command/PermissionsUserSetSuffixCommand.kt
package net.eduard.permissions.command

import net.eduard.api.lib.command.Command
import net.eduard.api.lib.command.PlayerOffline
import net.eduard.api.lib.command.Sender
import net.eduard.api.lib.modules.Mine
import net.eduard.permissions.api.PermissionsAPI
import net.eduard.permissions.core.PermsMessages

class PermissionsUserSetSuffixCommand :
    Command("setsuffix", "definirsufixo") {

    override fun onCommand(sender: Sender, args: List<String>) {
        val manager = PermissionsAPI.instance
        if (args.size < 4) {
            sendUsage(sender)
        } else {
            val nome = args[2]
            val suffixo = Mine.toChatMessage(args[3])
            val user = manager.getPlayer(PlayerOffline(nome))
            user.suffix = suffixo
            sender.sendMessage(
                PermsMessages.message("player-set-prefix")
                    .replace("\$player", "" + nome).replace("\$prefix", suffixo)
            )
            //
        }
    }


    init {
        usage = "/permissions user setsuffix <player> <permission>"
        description = "Definir o sufixo do jogador"
    }
}